.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,47 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
```

Comment:
unconfirmed_bug is probably more used and then we switch to bug once its reproduced

Thats how we do in t3-oss and trpc atleast